types/ramda/OTHER_FILES.txt

```diff
@@ -81,6 +81,7 @@ es/identical.d.ts
 es/identity.d.ts
 es/ifElse.d.ts
 es/inc.d.ts
+es/index.d.ts
```

Comment:
I'm not entirely sure why _this file_ is required to be here.

The other dist/src indexes are not required (in fact it is a test failure if included) because it gets picked up during tsconfig...

But this file _should_ be getting hit during the dynamic imports...
Similarly I would have expected the `es/package.json` file to be picked up since it impacts module resolution...

Oh well, this passes